gatsby-node.js

```diff
@@ -67,7 +67,6 @@ exports.createPages = async ({ graphql, actions }) => {
       allButterPost(
         limit: 2
         sort: {order: DESC, fields: published}
```

Comment:
I think we should also remove the ordering since the API returns already ordered posts